qa/L0_shared_memory/test.sh

```diff
@@ -62,32 +63,34 @@ for i in \
         fi
 
         export CLIENT_TYPE=$client_type
-        echo "Test: $i, client type: $client_type" >>$CLIENT_LOG
+        TMP_CLIENT_LOG="./tmp_client.log"
+        echo "Test: $i, client type: $client_type" >>$TMP_CLIENT_LOG
 
         set +e
-        python $SHM_TEST SharedMemoryTest.$i >>$CLIENT_LOG 2>&1
+        python $SHM_TEST SharedMemoryTest.$i >>$TMP_CLIENT_LOG 2>&1
```

Comment:
question: why do we need to use a different log file?

Reply:
This way we can echo the failing part of the client test that is failing and not _all_ tests that have been run so far.

Reply:
but the `TMP_CLIENT_LOG` contains the results from all of the clients as far as I can tell from your code. We're appending to the same file log.

Reply:
we remove the `$TMP_CLIENT_LOG` at L84 after we copy to the `$CLIENT_LOG`